Experimental-ckb-cli.md

```diff
@@ -0,0 +1,67 @@
+## Experimental CKB-CLI features
+A few of the examples in the README.md reflect a version of ckb-cli that is not the one contained in `nix/dep/ckb-cli/git.json`. For
+using the ledger with the version contained in this repo, see the following instructions.
+
+Note that the experimental features on Obsidian's fork of the ckb-cli client should only be used for development and testing purposes.
+
+### Message Signing
+
+To sign the hash of a message with their ledger a user may do the following (Note the recoverable flag is optional):
+
+```sh
+CKB> util sign-message --message "hello world i love nervos" --from-account <my-ledger-account> --recoverable
+message-hash: <blake2b hash of: magic_bytes + message>
+recoverable: false
+signature: <signature>
+```
+If a message is longer than 64 characters the ledger will display the first 61 chars, followed by an ellipsis (`...`)
+The ckb-cli accepts utf8 strings in its `--utf8-string` argument, but the ledger can not display all chars. If the ledger comes accross a
+character that it is unnable to display it will display an asterisk (`\*`) instead.
+
+Prompts on the Ledger device are as follows:
+
+|  Prompt 1 	|   Prompt 2  	|
+|:---------:	|:-----------:	|
+|   `Sign`  	|  `Message`  	|
+| `Message` 	| `<message>` 	|
+
+One can verfiy the signature as follows:
+
+```bash
+CKB> util verify-signature --message "hello world i love nervos" --from-account <my-ledger-account> --signature <signature from above>
+pubkey: <pubkey of my ledger's account root>
+recoverable: <signature type>
+verify-ok: true
+```
+
+A user may also switch out the `--message` option with a `--binary-hex` option, and the ledger will display
+the message as hex, instead of ascii characters
+```
+CKB> util sign-message --binary-hex '0x61' --from-account <my-ledger-account> 
+```
+
+### Signing a Hash ###
+The ledger also allows for the signing of a pre-hashed message. Due to the potential security risks of this option, a user must "opt-in" to this
+setting by going into the `Configuration` section of the app, and setting `Allow sign hash` to `On`.
+
+Note that the current implementation of the `ledger-sign-hash` util has a verification system built-in, and, as such, only accepts
```

Comment:
Can we expand on this point to mention why the existing `verify-signature` command will not work?